backend/src/database/sql/connection.ts

```diff
@@ -0,0 +1,24 @@
+import { Sequelize } from '@sequelize/core';
+import { env } from '../../config';
+import { createAssociations } from './models';
+import { Author, Book } from './models';
+
+export const sequelize = new Sequelize({
+  dialect: 'postgres',
+  schema: 'public',
+  models: [Author, Book],
+  url: env.PG_URL
+});
+
+createAssociations();
+
+export const establishSequelizeConnection = async () => {
+  try {
+    await sequelize.authenticate();
+    console.log('Connection has been established successfully.');
+  } catch (error) {
+    console.error('Unable to connect to the database:', error);
```

Comment:
In error handling, we need to consider if it is ok to continue. For example, here after logging the error, the app will start and try to serve requests even if it is not connected to DB. Worth considering if it should crash instead since it cannot do anything useful anyway.